### ISSUE TITLE
[Lab Submission] the check boxes in the prereqs render poorly in the workbench + are an accessibility flag bc they're unlabeled "form elements" these should be removed

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#439]
From https://github.com/carpentries-lab/reviews/issues/25#issuecomment-1787189928
